### PR TITLE
Feat/env folders copy

### PR DESCRIPTION
**Pull request by quinton11**
# Description 📣

- Web UI - allows copying of folders from a  select environment and path - [See](https://github.com/Infisical/infisical/issues/926)

## New
- An endpoint to handle creating batch folders

### CODE REVIEW

**Comment by quinton11** [comment]
@maidul98 @akhilmhdh 

Demo: 
https://github.com/Infisical/infisical/assets/70300837/19395034-9504-4215-a1c3-0ec5e3e07524

File: package.json [hunk 1]
```diff
@@ -21,5 +21,8 @@
 	"devDependencies": {
 		"eslint": "^8.29.0",
 		"husky": "^8.0.3"
+	},
+	"dependencies": {
+		"@radix-ui/react-tabs": "^1.0.4"
```
Comment (ap0h) [inline package.json:26]: **question:** Shouldn't this be in the `/frontend/package.json`

Comment (quinton11) [reply package.json:26]: Right, must've run the install pkg cmd in the root directory

**Comment by maidul98** [comment]
This PR is now marked stale. The codebase as changed drastically due to Postgres migration.

**Pull request closed without merging**